docs/audit-log-filter/reading-audit-log-filter-files.md

```diff
@@ -0,0 +1,6 @@
+# Reading Audit Log Filter files
```

Comment:
Is there a plan to extend this section with more detailed description of UDFs for logs reading and some examples?

Reply:
Will investigate.